tillerinobot/src/main/java/tillerino/tillerinobot/lang/Spanish.java

```diff
@@ -0,0 +1,217 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.IRCBot.IRCBotUser;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+
+/**
+ * @author Sibchil https://osu.ppy.sh/u/3988045 https://github.com/sibchil
+ */
+public class Spanish implements Language {
+
+	@Override
+	public String unknownBeatmap() {
+		return "Lo siento, no conozco ese mapa. Es muy reciente, muy dif�cil, unranked o modo de osu! no est�ndar.";
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Ehh... Parece que Tillerino humano estrope� mi cableado."
+		                + " Si no se da cuenta pronto, �podr�as [https://github.com/Tillerino/Tillerinobot/wiki/Contact informarle]? (c�digo de referencia "
+		                + marker + ")";
+	}
+
+	@Override
+	public String externalException(String marker) {
+		return "�Qu� est� pasando? Solo recibo disparates del servidor de osu. �Podr�as decirme que se supone que significa esto? 0011101001010000"

```

Comment:
TL;DR: "decir" is more commonly used to refer to what a person says while "significa" is used in the same way as "mean".

The english version says "Can you tell me what this is supposed to mean?". I think it's better "significa" from "significado" which is the same as (verb) "means" from "meaning". In Spanish we usually say "¿Qué significa?" as in what does it mean, what does it represent. We use the expression "¿Qué quieres decir?" in the same way as English "What do you mean?" but refering to a person because "decir" means "say". The more or less literatal translation from "¿Qué quieres decir?" to English would be "What are you trying to say?". Now the 0011101001... etc binary is received from a server so it is not personal so "¿Qué significa esto?" is more appropiate.